manifests/cf-manifest/operations.d/730-elasticache-broker.yml

```diff
@@ -71,6 +71,12 @@
                       free: true
                       metadata:
                         displayName: Redis Non-clustered Tiny
+                    - id: b6949ea7-5c98-4c69-b981-4b5318ea8b7c
+                      name: medium-clustered-3.2
+                      description: 6.37GB RAM, clustered (1 shard), 3 nodes, failover, daily backups
```

Comment:
Do we need 3 nodes, or would 2 be sufficient? This will presumably have a cost impact.

Reply:
It will have a cost impact, though I'm happy with 3 as the request has been about both size and failover capability.

Reply:
I was thinking that 2 should be enough for failover (with RDS the HA instances only have 2 nodes, I was expecting this would be similar)

Reply:
3 prevents a split brain scenario in the event of network partitioning though.